husky_extensions/util.js

```diff
@@ -19,53 +19,9 @@ define(['services/husky/util'], function(Util) {
         name: 'Util',
 
         initialize: function(app) {
-            app.core.util.compare = Util.compare;
-
-            app.core.util.typeOf = Util.typeOf;
-
-            app.core.util.isEqual = Util.isEqual;
-
-            app.core.util.isEmpty = Util.isEmpty;
-
-            app.core.util.foreach = Util.foreach;
-
-            app.core.util.load = Util.load;
-
-            app.core.util.save = Util.save;
-
-            app.core.util.cropMiddle = Util.cropMiddle;
-
-            app.core.util.cropFront = Util.cropFront;
-
-            app.core.util.cropTail = Util.cropTail;
-
-            app.core.util.contains = Util.contains;
-
-            app.core.util.isAlphaNumeric = Util.isAlphaNumeric;
-
-            app.core.util.uniqueId = Util.uniqueId;
-
-            app.core.util.delay = Util.delay;
-
-            app.core.util.union = Util.union;
-
-            app.core.util.deepCopy = Util.deepCopy;
-
-            app.core.util.getParameterByName =  Util.getParameterByName;
-
-			app.core.util.template = Util.template;
-
-            app.core.util.escapeHtml = Util.escapeHtml;
-
-            app.core.util.arrayGetColumn = Util.arrayGetColumn;
-
-            app.core.util.removeFromArray = Util.removeFromArray;
-
-            app.core.util.capitalizeFirstLetter = Util.capitalizeFirstLetter;
-
-            app.core.util.arrayMap = Util.arrayMap;
-
-            app.core.util.object = Util.object;
+            for (var key in Util) {
+                app.core.util[key] = Util[key];
+            }
```

Comment:
no because the app.core.util is an object which is already initialized from aura and has a few functions from there. so i have to extend this object.